crm/modules/user/user.inc.php

```diff
@@ -161,18 +161,13 @@ function user_install ($old_revision = 0) {
                 foreach ($default_perms[$role] as $perm) {
                     $esc_perm = mysqli_real_escape_string($db_connect, $perm);
                     $sql = "
-                        SELECT * FROM `role_permission`
```

Comment:
So instead of checking if the role is there then only inserting if not, it will just insert it anyway?  Won't this cause duplicates if ran more than once?  Can you guarantee this will only get executed once?

Reply:
@ramgarden yeah, because it's aprt of the install routine for the user module, it should only run on initial deployment of a new instance, or when upgrading from a very old version, & would only run once as it's in this specific version of the user module